app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
```

Comment:
- [ ] Hi team, the console error on the screenshot below hinders me from setting up your project completely. Hence I couldn't register or login into your app. If the error is from my end, I'll be happy to connect with you guys to resolve else make sure to fix it and your app should be working as intended. 


![Screenshot from 2022-08-24 19-37-48](https://user-images.githubusercontent.com/63087491/186498650-49c6cd59-a2b5-4e54-a81e-aa421b13d072.png)



Thanks :pray:

Reply:
Hello @FrederickMih 

Thank you for your review and recommendations. We tested the project in both platforms (Windows and Ubuntu) and the problem is related with the bootsnap gem as follows:
- In order for us to be able to run the project successfully in windows platform we have to change the bootsnap gem to `gem 'bootsnap', '>= 1.1.0', '< 1.4.2', require: false`.
- In order for us to be able to run the project successfully in Ubuntu platform we have to change the bootsnap gem to `gem "bootsnap", "~> 1.13", require: false`.

As you are using Ubuntu try installing the last gem, with the following commands:
```
bundle remove bootsnap
bundle add bootsnap
```
IMPORTANT: After installing the gem you should write `require:false` at the end of the line of bootsnap gem.

Reply:
Hi @jrosario19 .


Thanks 🙏 so much